packages/roosterjs-content-model/lib/modelApi/image/applyImageBorderFormat.ts

```diff
@@ -0,0 +1,50 @@
+import { Border, extractBorderValues } from '../../domUtils/borderValues';
+import { ContentModelImage } from '../../publicTypes/segment/ContentModelImage';
+import { ptToPx } from '../../formatHandlers/utils/pointsToPixels';
+
+/**
+ * @internal
+ */
+export default function applyImageBorderFormat(
+    image: ContentModelImage,
+    border: Border,
+    isPt?: boolean
+) {
+    const format = image.format;
+    const { width, style, color } = border;
+    const borderKey = 'borderTop';
```

Comment:
Why always top? Should we just retrieve from each side of existing values?

Reply:
We cannot support different styles for each border, then we just need to check one border

Reply:
hmm ok. In most cases it should be fine. Just a little bit concern about some corner case such as we paste some image that has different borders in each side. We can solve it when that really happens